md_api/src/main/java/com/wipro/ats/bdre/md/api/Lineage.java

```diff
@@ -45,6 +45,10 @@ public Lineage() {
     @Autowired
     LineageRelationDAO lineageRelationDAO;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)